docs/book/src/introduction/installation.md

```diff
@@ -2,35 +2,91 @@
+   npacking and moving forc-doc to /Users/user/.fuelup/toolchains/beta-2-aarch64-apple-darwin/bin
+   Unpacking and moving forc to /Users/user/.fuelup/toolchains/beta-2-aarch64-apple-darwin/bin
+   Unpacking and moving forc-deploy to /Users/user/.fuelup/toolchains/beta-2-aarch64-apple-darwin/bin
+   Unpacking and moving forc-run to /Users/user/.fuelup/toolchains/beta-2-aarch64-apple-darwin/bin
+   Unpacking and moving forc-lsp to /Users/user/.fuelup/toolchains/beta-2-aarch64-apple-darwin/bin
+   Unpacking and moving forc-fmt to /Users/user/.fuelup/toolchains/beta-2-aarch64-apple-darwin/bin
+   Fetching core forc dependencies
+   Installed forc v0.31.1 for toolchain 'beta-2-aarch64-apple-darwin'
+
+   Adding component forc-explore v0.28.1 to 'beta-2-aarch64-apple-darwin'
+   Fetching binary from https://github.com/FuelLabs/forc-explorer/releases/download/v0.28.1/forc-explore-0.28.1-aarch64-apple-darwin.tar.gz
+   Unpacking and moving forc-explore to /Users/user/.fuelup/toolchains/beta-2-aarch64-apple-darwin/bin
+   Installed forc-explore v0.28.1 for toolchain 'beta-2-aarch64-apple-darwin'
+
+   Adding component forc-wallet v0.1.2 to 'beta-2-aarch64-apple-darwin'
+   Fetching binary from https://github.com/FuelLabs/forc-wallet/releases/download/v0.1.2/forc-wallet-0.1.2-aarch64-apple-darwin.tar.gz
+   Unpacking and moving forc-wallet to /Users/user/.fuelup/toolchains/beta-2-aarch64-apple-darwin/bin
+   Installed forc-wallet v0.1.2 for toolchain 'beta-2-aarch64-apple-darwin'
+
+   Adding component fuel-core v0.15.3 to 'beta-2-aarch64-apple-darwin'
+   Fetching binary from https://github.com/FuelLabs/fuel-core/releases/download/v0.15.3/fuel-core-0.15.3-aarch64-apple-darwin.tar.gz
+   Unpacking and moving fuel-core to /Users/user/.fuelup/toolchains/    beta-2-aarch64-apple-darwin/bin
+   Installed fuel-core v0.15.3 for toolchain 'beta-2-aarch64-apple-darwin'
+
+   Adding component fuel-indexer v0.1.13 to 'beta-2-aarch64-apple-darwin'
+   Fetching binary from https://github.com/FuelLabs/fuel-indexer/releases/download/v0.1.13/fuel-indexer-0.1.13-aarch64-apple-darwin.tar.gz
+   Unpacking and moving fuel-indexer to /Users/user/.fuelup/toolchains/beta-2-aarch64-apple-darwin/bin
+   Installed fuel-indexer v0.1.13 for toolchain 'beta-2-aarch64-apple-darwin'
+
+   Installed:
+   - forc 0.31.1
+   - forc-explore 0.28.1
+   - forc-wallet 0.1.2
+   - fuel-core 0.15.3
+   - fuel-indexer 0.1.13
+
+   The Fuel toolchain is installed and up to date
+   ```
+
+You're all set to start building!
 
-to install the latest Fuel toolchain.
+#### Need Help?
 
-You can run the same command at a later time to update the toolchain.
+Check out the tooling section in the [Fuel forum](https://forum.fuel.network/) if you're running into problems through the installation process. If you don't see your question, post the issue you're running into with as many details as possible and the team will get back to you asap!
```

Comment:
There probably should also be a link to the fuelup book here: https://fuellabs.github.io/fuelup/latest/, so that people know where to look to self-troubleshoot issues before posting on the forum

```suggestion
You may refer to [The Fuelup Book](https://fuellabs.github.io/fuelup/latest/) for an in-depth look into fuelup, or check out the tooling section in the [Fuel forum](https://forum.fuel.network/) if you're running into problems through the installation process. If you don't see your question, post the issue you're running into with as many details as possible and the team will get back to you asap!
```